client/my-sites/media-library/content.jsx

```diff
@@ -358,6 +358,14 @@ export class MediaLibraryContent extends React.Component {
 			return this.renderConnectExternalMedia();
 		}
 
+		const listKey = [
+			'list-',
```

Comment:
Nit: we can remove the trailing `-` here, because we'll get another one coming from the `join( '-' )` below.

Reply:
You're right! I'll add that and then :shipit: